docs/streams/developer-guide/app-reset-tool.html

```diff
@@ -65,6 +65,10 @@
         </dl>
         <div class="section" id="step-1-run-the-application-reset-tool">
             <h2>Step 1: Run the application reset tool<a class="headerlink" href="#step-1-run-the-application-reset-tool" title="Permalink to this headline"></a></h2>
+            <p>If you are using <strong>AK 4.1 or earlier</strong>, run the classic application reset tool as follows,
+                but if you are on <strong>AK 4.2+ (Streams Rebalance Protocol enabled)</strong>, use the <a href = "">Streams groups CLI</a> instead.</p>
+            <p>If you are using <strong>AK 4.1 or earlier</strong>, run the classic application reset tool. If you are on <strong>AK 4.2+ with Streams Rebalance Protocol enabled</strong>, use the <a href="kafka-streams-group-sh.html#delete-offsets">Streams groups CLI</a> instead to delete the offsets.</p>
```

Comment:
Seems we have the same sentence twice?

I would put "classic" vs "streams" in the core of the distinction and not the users versions -- atm the text put the used version in the center, what might not be ideal. More like:
```
If you are using the classic rebalance protocol use [...].
If you are using the stream rebalance protocol (available since AK 4.2) use [...]
```
(or something like this)